src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/LanguageServices/SemanticsFactsService/AbstractSemanticFactsService.cs

```diff
@@ -97,6 +97,31 @@ public SyntaxToken GenerateUniqueName(string baseName, IEnumerable<string> usedN
                     baseName, usedNames, this.SyntaxFacts.IsCaseSensitive));
         }
 
+#nullable enable
+
+        protected static IMethodSymbol? FindDisposeMethod(Compilation compilation, ITypeSymbol? type, bool isAsync)
```

Comment:
💭 this might be a good use case for partial classes where nullable enabled methods can live

Reply:
yeah.  my goal is just to NRT enable this fully anyways.  just haven't gotten around to it yet :)